docs/platforms/python/common/metrics/index.mdx

```diff
@@ -95,3 +100,122 @@ To emit a timer, do the following:
 with sentry_sdk.metrics.timing(key="event_processing_time"):
 	process()
 ```
+
+
+## Options
+
+You do not need to do anything to enable metrics support. There are some options you can set/change to influence the metrics behavior. As Metrics is still in the beta phase, those options are under the `_experimental` option.
+
+<ConfigKey name="metrics_summary_sample_rate">
+
+The sampling rate for metrics summaries. Float value that defaults to `1.0`.
+
+```python
+sentry_sdk.init(
+	...
+    _experiments={
+		"metrics_summary_sample_rate": 0.5,
+	}
+)
+```
+
+</ConfigKey>
+
+<ConfigKey name="should_summarize_metric">
+
+A callback function hat is called with a metric `key` and the `tags` associated with that key. If the callback function returns `True` a summary for the metrics with the given `key` is generated, if the callback function returns `False` no summary of the metric with the given `key` is generated.
+
+In this example only summaries for the metrics with the key `important.metric.key` will be generated:
+```python
+def should_summarize_metric(key, tags):
+	return key == "important.metric.key"
+
+sentry_sdk.init(
+	...
+    _experiments={
+		"should_summarize_metric": should_summarize_metric,
+	}
+)
+```
+
+</ConfigKey>
+
+<ConfigKey name="before_emit_metric">
+
+A callback function that is called before a metric is emmited. If the callback returns `False` the metric is not emitted, if the callback returns `True` the metric is emitted. The given tags can also be updated in the callback function.
+
+```python
+def before_emit(key, tags):
+	if key == "removed-metric":
+		return False
+
+	tags["extra"] = "foo"
+	del tags["release"]
+
+	return True
+
+sentry_sdk.init(
+	...
+    _experiments={
+		"before_emit_metric": before_emit,
+	}
+)
+```
+
+</ConfigKey>
+
+<ConfigKey name="metric_code_locations">
+
+If `True` the line of code where the metric was emitted will be added to the metric. The code location will be omitted when this option is set to `False`.
```

Comment:
```suggestion
If `True`, the line of code where the metric was emitted will be added to the metric. If `False`, the code location will be omitted.
```
Would be helpful to add a line saying what the default value is here.

Reply:
done